echopype/convert/parse_azfp.py

```diff
@@ -112,9 +112,14 @@ def compute_temp(counts):
             and the counts from ancillary
             """
             v_in = 2.5 * (counts / 65535)
+            # Occurs when temperature sensor is not active (eg, glider deployments)
+            if self.parameters["kc"] - v_in <= 0:
+                return np.nan
+
```

Comment:
`self.parameters["kc"] - v_in <= 0` is fairly arbitrary since v_in is essentially trying to get some sensor voltage values.

Since in the case of the glider, all temperature-related parameters in `self.parameters` are 0, it is likely safer to test for these cases?

Or, alternatively, test for the case when `R<=0` so that we don't go into log(negative values).

Reply:
Are we certain that the relevant temperature parameters (`ka`, `kb`, `kc`) will always be 0 in circumstances like the Rutgers glider where the temperature sensor is turned off (or not deployed?). Do you know if it's documented that that will always be the case? Assuming that is true, can we reduce the 0 tests to just one of the parameters, to minimize the footprint of the test? 

BTW, `v_in` values are small (and apparently constant) but non-zero. See https://github.com/OSOceanAcoustics/echopype/issues/198#issuecomment-791560031

I'll comment on `compute_tilt` below your comment.

Reply:
Addressed in a new commit, for both `compute_temp` and `compute_tilt`.